Gopkg.toml

```diff
@@ -24,6 +24,15 @@
 #   go-tests = true
 #   unused-packages = true
 
+required = [
+ "k8s.io/apiserver/pkg/endpoints/request"
+]
+
+# Fix:
+#   vendor/k8s.io/kubernetes/pkg/util/parsers/parsers.go:36:16: undefined: reference.ParseNormalizedNamed
```

Comment:
Found this workaround in a istio repo, any better idea how I can fix this?

Reply:
We do this in other places as well. These errors occur in pairs as it seems.